tests/e2e/interOp/validation_of_operating_modes/bridge_mode/country_code_channel_division/country_code_GB/iOS/test_20mhz.py

```diff
@@ -0,0 +1,1773 @@
+from logging import exception
+import unittest
+import warnings
+from perfecto.test import TestResultFactory
+import pytest
+import sys
+import time
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from appium import webdriver
+from selenium.common.exceptions import NoSuchElementException
+import random
+import string
+import sys
+import allure
+
+if 'perfecto_libs' not in sys.path:
+    sys.path.append(f'../libs/perfecto_libs')
+
+pytestmark = [pytest.mark.interop, pytest.mark.ios, pytest.mark.interop_ios, pytest.mark.interop_country_codeGBI1,
+               pytest.mark.interop_uc_sanity, pytest.mark.bridge]
+
+
+from iOS_lib import closeApp, openApp, get_WifiIPAddress_iOS, ForgetWifiConnection, ping_deftapps_iOS, \
+    Toggle_AirplaneMode_iOS, set_APconnMobileDevice_iOS, verify_APconnMobileDevice_iOS, Toggle_WifiMode_iOS, tearDown,\
+    verifyUploadDownloadSpeediOS, get_ip_address_ios
+
+
+setup_params_general1 = {
+    "mode": "BRIDGE",
+    "ssid_modes": {
+        "wpa2_personal": [{"ssid_name": "ssid_wpa2_2g", "appliedRadios": ["2G"], "security_key": "something"},
+                          {"ssid_name": "ssid_wpa2_5g", "appliedRadios": ["5G"], "security_key": "something"}
+                          ]},
+    "rf": {"5G":
+        {'band': '5G',
+        'country': 'GB',
+        "channel-mode": "VHT",
+        'channel-width': 20,
+        "channel": 36},
+        "2G":
+        {'band': '2G',
+        'country': 'GB',
+        "channel-mode": "VHT",
+        'channel-width': 20,
+        "channel": 1}
+           },
+    "radius": False
+}
+
+for sec_modes in setup_params_general1['ssid_modes'].keys():
+    for i in range(len(setup_params_general1['ssid_modes'][sec_modes])):
+        N = 3
+        rand_string = (''.join(random.choices(string.ascii_uppercase +
+                                     string.digits, k=N)))+str(int(time.time_ns())%10000)
+        setup_params_general1['ssid_modes'][sec_modes][i]['ssid_name'] = setup_params_general1['ssid_modes'][sec_modes][i]['ssid_name'] + "_"+ rand_string
+
+# @allure.suite(suite_name="interop country code")
+# @allure.sub_suite(sub_suite_name="Bridge Mode country code")
+# @pytest.mark.InteropsuiteA
+# @allure.feature("BRIDGE MODE CLIENT CONNECT")
```

Comment:
Don't comment these lines and remove the marker in them
```suggestion
@allure.suite(suite_name="interop country code")
@allure.sub_suite(sub_suite_name="Bridge Mode country code")
@allure.feature("BRIDGE MODE CLIENT CONNECT WITH COUNTRY CODE")
```